### PR TITLE
Remove the unused `final` keyword on local variables and repackaging for the `testlib` module.

### PR DESCRIPTION
**Pull request by illia-shepilov**
This PR provides the following changes for the `testlib` module:

- The `final` keyword on local variables was removed.
- The package was renamed from`io.spine.test` to `io.spine.testing`.

### CODE REVIEW

**Comment by illia-shepilov** [comment]
@armiol PTAL.

File: testlib/src/main/java/io/spine/testing/TempDirectory.java [hunk 1]
```diff
@@ -1,14 +1,24 @@
 /*
- * Copyright 2015-2018 the original author or authors.
+ * Copyright 2018, TeamDev. All rights reserved.
```
Comment (armiol) [inline testlib/src/main/java/io/spine/testing/TempDirectory.java]: Please revert the changes to this header.
